tests/test_dice_interface/test_dice_genetic.py

```diff
@@ -112,6 +112,37 @@ def test_predict_custom(self, desired_class, sample_custom_query_2, mocker):
         assert custom_preds[0] == desired_class
 
 
+class TestDiceGeneticBinaryStrClassificationMethods:
+    @pytest.fixture(autouse=True)
+    def _initiate_exp_object(self, genetic_binary_str_classification_exp_object):
+        self.exp = genetic_binary_str_classification_exp_object  # explainer object
+
+    # When invalid desired_class is given
+    @pytest.mark.parametrize(("desired_class", "total_CFs"), [(7, 3)])
+    def test_no_cfs(self, desired_class, sample_custom_query_1, total_CFs):
+        with pytest.raises(UserConfigValidationException):
```

Comment:
Could you test the exception message encountered here?

Reply:
Done

Reply:
@amit-sharma have you sent an update commit on this comment?

Reply:
oh yes, pushed now. Also added the multiclass test case.

Reply:
Thank you